routers/web/repo/lfs.go

```diff
@@ -461,22 +462,29 @@ func LFSPointerFiles(ctx *context.Context) {
 					// Can we fix?
 					// OK well that's "simple"
 					// - we need to check whether current user has access to a repo that has access to the file
-					result.Accessible, err = models.LFSObjectAccessible(ctx.User, pointerBlob.Oid)
+					result.Associatable, err = models.LFSObjectAccessible(ctx.User, pointerBlob.Oid)
 					if err != nil {
 						return err
 					}
-				} else {
-					result.Accessible = true
+					if !result.Associatable {
+						associated, err := models.LFSObjectIsAssociated(pointerBlob.Oid)
+						if err != nil {
+							return err
+						}
+						result.Associatable = !associated
+					}
```

Comment:
I don't understand why this is necessary

Reply:
What do you mean about `this`? 

If `this` means this PR: it just adds a feature to help users to fix their broken LFS (eg: database was lost, table was truncated)

If `this` means the code block: I think it's better to make sure that there is no record in database. If the LFS oid already exists in database table, then the oid should have been associated to other repositories, then we need the `LFSObjectAccessible` above to check if the user can access it.

I am still not sure what's the best way to do the associating, so suggestions are welcome.